PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -1017,12 +1017,33 @@ void TabBarPlus::drawItem(DRAWITEMSTRUCT *pDrawItemStruct, bool isDarkMode)
 	{
 		::MessageBox(NULL, TEXT("! TCM_GETITEM"), TEXT(""), MB_OK);
 	}
+
+	const COLORREF colorActiveBg = isDarkMode ? NppDarkMode::getSofterBackgroundColor() : ::GetSysColor(COLOR_BTNFACE);
+	const COLORREF colorInactiveBgBase = isDarkMode ? NppDarkMode::getBackgroundColor() : grey;

```

Comment:
@donho 
> There's a visual glitch in this PR: ![image](https://user-images.githubusercontent.com/90293/202172205-6e4d977d-85b8-4e1b-878e-8e0a983f3bd6.png)
> 
> Do you know what does make this glitch?

replace `grey` with `::GetSysColor(COLOR_BTNFACE)`

Reply:
Yes, I've fixed that, but I can't push my modification into your branch remotely (from my local cloned repository).
```
don@xxxxxxx MINGW64 /c/XXXXX/notepad-plus-plus (ozone10-DarkModeNoThemeIconChange)
$ git push --set-upstream origin ozone10-DarkModeNoThemeIconChange
Enumerating objects: 132, done.
Counting objects: 100% (132/132), done.
Delta compression using up to 16 threads
Compressing objects: 100% (54/54), done.
Writing objects: 100% (115/115), 31.81 KiB | 2.65 MiB/s, done.
Total 115 (delta 100), reused 75 (delta 61), pack-reused 0
remote: Resolving deltas: 100% (100/100), completed with 16 local objects.
remote:
remote: Create a pull request for 'ozone10-DarkModeNoThemeIconChange' on GitHub by visiting:
remote:      https://github.com/notepad-plus-plus/notepad-plus-plus/pull/new/ozone10-DarkModeNoThemeIconChange
remote:
To https://github.com/notepad-plus-plus/notepad-plus-plus.git
 * [new branch]        ozone10-DarkModeNoThemeIconChange -> ozone10-DarkModeNoThemeIconChange
Branch 'ozone10-DarkModeNoThemeIconChange' set up to track remote branch 'ozone10-DarkModeNoThemeIconChange' from 'origin'.
```
made a new PR:
https://github.com/notepad-plus-plus/notepad-plus-plus/pull/12527

Do you have any idea about the correct command to push my commit into this PR?

Reply:
@donho 
did you use `git fetch` or ``git pull` to get my latest commit

Reply:
@donho 
maybe this
```
git clone https://github.com/notepad-plus-plus/notepad-plus-plus.git
git remote add ozone10pr20221116 https://github.com/ozone10/notepad-plus-plus.git && git fetch ozone10pr20221116
git checkout -b ozone10-DarkModeNoThemeIconChange ozone10pr20221116/DarkModeNoThemeIconChange

"do some changes"

git add
git commit
git push ozone10pr20221116 HEAD:DarkModeNoThemeIconChange
```

Reply:
@ozone10 
Terrific! Thank you for help!